data-raw/simulate_multiple_phenotypes.R

```diff
@@ -0,0 +1,101 @@
+set.seed(982348)
+library(dplyr)
+library('MASS')
+
+load('data/random_genotype_matrix.rda')
```

Comment:
What is the source of this matrix?

Reply:
I did not want to include the WTCCC data in the repository so I created a random matrix just as a substitute. It is not a realistic simulation, just entirely random.